modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm

```diff
@@ -9,13 +9,13 @@
 	obj_flags = UNIQUE_RENAME
 
 /obj/item/ttsdevice/attack_self(mob/user)
-	user.balloon_alert_to_viewers("Typing on [initial(name)].", "You begin typing.")
+	user.balloon_alert_to_viewers("typing on [initial(name)]...", "started typing...")
```

Comment:
```suggestion
	user.balloon_alert_to_viewers("starts typing...", "started typing...")
```
Actually I'm not sure the name of the tts device is very good to have in a balloon alert